core/interfaces.go

```diff
@@ -11,9 +11,9 @@ import (
 	"net/http"
 	"time"
 
-	jose "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
```

Comment:
Your rule sounds like a good one to me. I might add a (3) for when the default package name creates potential confusion with local variable names (e.g. we use import CFSSL's config package as `cfsslConfig` in `ca`, to distinguish it from the CA's config object.